Rails models

Dmitry173 opened the pull request.

Dmitry173 committed:
Rails models

BubuntuClu commented on db/migrate/20210224143056_add_reference_index_to_all_tables.rb:
```diff
@@ -0,0 +1,7 @@
+class AddReferenceIndexToAllTables < ActiveRecord::Migration[5.1]
+  def change
+    add_reference(:tests, :categories, foreign_key: true)
+    add_reference(:questions, :tests, foreign_key: true)
```
t.integer "tests_id"
у тебя сгенерилось такое поле, во множественном числе, из коробки это не будет работать.

перенси создание этих связей в миграцию создания самой таблицы, зачем плодить файлы.
вообще, в идеале, тут должно быть всего 5 миграций. по одной на каждую таблицу, в рамках которой ты создаешь все нужные столбцы и ограничения на них

Dmitry173 committed:
Changes to be committed:
	modified:   db/migrate/20210224142844_create_tests.rb
	modified:   db/migrate/20210224142850_create_categories.rb
	modified:   db/migrate/20210224142855_create_questions.rb
	modified:   db/migrate/20210224142909_create_answers.rb
	modified:   db/migrate/20210224142913_create_users.rb
	deleted:    db/migrate/20210224143056_add_reference_index_to_all_tables.rb
	deleted:    db/migrate/20210224143102_add_tests_level_default.rb
	deleted:    db/migrate/20210224143106_add_to_all_tables_null_constraints.rb
